lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -622,6 +622,15 @@ func (c *Client) startInputStreaming(stopCh chan struct{}) error {
 					return trace.Errorf("SharedDirectoryMoveResponse failed: %v", errCode)
 				}
 			}
+		case tdp.SharedDirectoryTruncateResponse:
```

Comment:
Do we need some backwards compat logic here, or is the thought that we will never send a response to an old w_d_s that doesn't understand it because said w_d_s would have never sent the initial request?

Reply:
> we will never send a response to an old w_d_s that doesn't understand it because said w_d_s would have never sent the initial request?

Yes, only a w_d_s that sends a truncate request will ever receive a truncate response, and all w_d_s's that send truncate requests can handle truncate responses.